lib/facter/util/wmi.rb

```diff
@@ -2,7 +2,10 @@ module Facter::Util::WMI
   class << self
     def connect(uri = wmi_resource_uri)
       require 'win32ole'
-      WIN32OLE.connect(uri)
+      #WIN32OLE.connect(uri)
+      swbem = WIN32OLE.new("WbemScripting.SWbemLocator")
```

Comment:
Filed https://tickets.puppetlabs.com/browse/FACT-1418 to cover this as we'll have to think a little about breaking API implications.